### PR TITLE
slight visual improvements on the group wrapper

### PR DESCRIPTION
**Pull request by its-mahdi**
showing groups of records again, with slight visual improvements.

### CODE REVIEW

File: src/styles/ResultsTable.css [hunk 1]
```diff
@@ -106,7 +106,7 @@ tbody {
 }
 
 .group-wrapper {
-  border-bottom: 5px solid #cbcce0;
+  border-bottom: 2px solid #e0e0e0;;
```
Comment (xwatkins) [inline src/styles/ResultsTable.css]: use rem for consistency

File: src/ui/components/ResultsTable.jsx [hunk 2]
```diff
@@ -70,23 +71,35 @@ class ResultsTable extends Component {
         return (a.alignment_difference - b.alignment_difference);
       });
 
-    return rows.slice(0, 1).map((mapping) => {
+    // return rows.slice(0, 1).map((mapping) => {
+    return rows.map((mapping) => {
       const key = `${mapping.ensemblTranscript.enstId}_${mapping.uniprotEntry.uniprotAccession}`;
 
       // -- temp disabled for simplified table
```
Comment (xwatkins) [inline src/ui/components/ResultsTable.jsx]: Unnecessary comment

File: src/ui/components/ResultsTable.jsx [hunk 3]
```diff
@@ -70,23 +71,35 @@ class ResultsTable extends Component {
...
+        !mapping.uniprotEntry.isCanonical &&
+        wrapper.index !== this.state.expandGroupIndex)
+      {
+        return null;
+      }
 
       return (
         <Link to={`${BASE_URL}/mapping/${mapping.mappingId}`} key={key} className="table-row">
           <div className="table-cell">
             {/* temporarily disabled 'isoform' flag */}
```
Comment (xwatkins) [inline src/ui/components/ResultsTable.jsx]: Same

File: src/ui/components/ResultsTable.jsx [hunk 4]
```diff
@@ -118,6 +131,22 @@ class ResultsTable extends Component {
     });
   };
 
+  renderBorderWrapperRows = (group) => {
+    const { wrapper } = group;
+    const {
```
Comment (xwatkins) [inline src/ui/components/ResultsTable.jsx]: None of these are used

**Comment by its-mahdi** [comment]
Switched to 'rem' and removed the commented code. Fixed PropTypes and `key` warnings too. For 'unmapped' entries that's a different issue and we haven't agreed on any solution yet. That would be solved in its own PR.

**Pull request merged**